govcd/disk.go

```diff
@@ -380,3 +394,32 @@ func FindDiskByHREF(client *Client, href string) (*Disk, error) {
 	// Return the disk
 	return disk, nil
 }
+
+// Find vm using vApp name and VM name. Returns VMRecord query return type
+func (vdc *Vdc) QueryDisk(diskName string) (DiskRecord, error) {
+
```

Comment:
Validate if diskName is not empty as it's done in `func (vdc *Vdc) QueryVM(vappName, vmName string)`

Reply:
Fixed